tests/stub/errors/scripts/test_error.script

```diff
@@ -0,0 +1,13 @@
+!: BOLT 4.4
+
+A: HELLO {"{}": "*"}
+*: RESET
+C: RUN { "U": "*"} {"{}": "*"} {"{}": "*"}
+S: FAILURE {"code": "#ERROR_CODE#", "message": "Something went wrong"}
```

Comment:
I'm just doing this on run, but should it really be done in the block of a transaction?